glib/glib.go

```diff
@@ -308,7 +308,7 @@ func goMarshal(closure *C.GClosure, retValue *C.GValue,
 }
 
 // gValueSlice converts a C array of GValues to a Go slice.
-func gValueSlice(values *C.GValue, nValues int) (slice []C.GValue) {
+func gValueSlice(values *C.GValue, nValues int) (slice []*C.GValue) {
```

Comment:
I don't think this is correct. I get failures in the callbacks with `no suitable Go value for arg 0: invalid GValue`.
But I don't understand this code. Please, don't merge yet.